plugins/rollbar/lib/samson_rollbar/rollbar_user_informer.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SamsonRollbar
+  class RollbarUserInformer
+    ENV_KEY = "rollbar.exception_uuid"
+
+    class << self
+      attr_accessor :user_information_placeholder
+    end
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      status, headers, body = @app.call(env)
+
+      if self.class.user_information_placeholder.present? && error_uuid = env[ENV_KEY]
+        replacement = replacement(error_uuid.to_s)
+        body = replace_placeholder(replacement, body, headers)
+        headers["Error-Id"] = error_uuid.to_s
+      end
+      [status, headers, body]
+    end
+
+    private
+
+    # - body interface is .each so we cannot use anything else
+    # - always call .close on the old body so it can get garbage collected if it is a File
+    def replace_placeholder(replacement, body, headers)
+      new_body = []
+      body.each { |chunk| new_body << chunk.gsub(self.class.user_information_placeholder, replacement) }
+      headers["Content-Length"] = new_body.inject(0) { |sum, x| sum + x.bytesize }.to_s
+      new_body
+    ensure
+      body.close if body && body.respond_to?(:close)
+    end
+
+    def replacement(error_uuid)
```

Comment:
this needs to be a setting and not hard-coded like `Airbrake.user_information =`